docs/developer/extending/apps.mdx

```diff
@@ -48,29 +48,32 @@ If there are no errors in the response, a new app is created:
 ```
 
 The response contains the `authToken` field - this is the token that the app needs to include in requests to Saleor API to authorize itself and have access to queries or mutations that it needs. Apps can authorize in Saleor using [the Authorization header, just as regular users](developer/users.mdx#authentication).
+:::note
+App uses the same header to authorize but token should be provided with a different prefix. Use `Bearer <TOKEN>` instead of `JWT <TOKEN>` prefix.
+:::
 
 :::note
 The `authToken` field is only available in the mutation response and cannot be fetched later, except for the last four digits.
 :::
 
 ## Creating authorization keys
 
-Creating an app with `serviceAccountCreate` mutation creates the default authorization token (`authToken`). You can create more authorization keys with the `serviceAccountTokenCreate`. The mutation takes the following fields:
+Creating an app with `appCreate` mutation creates the default authorization token (`authToken`). You can create more authorization keys with the `appTokenCreate`. The mutation takes the following fields:
 
-- `serviceAccount`: ID of the app.
+- `app`: ID of the app.
 - `name`: (optional) name of the key.
 
 ```graphql
```

Comment:
You can highlight important lines in the examples by putting their numbers or ranges in curly braces next to the syntax marker. We do that in other sections of the docs, for example `graphql {2-4}`.